src/components/shared/styles/form.scss

```diff
@@ -24,8 +28,12 @@
     font-size: $mediumFontSize;
 }
 
-.sectionDivider {
-    padding-top:20px;
-    border-bottom: 1.5px solid $lightGrey;
-    width: 100%;
-}
+.section {
+    margin: 30px 0;
```

Comment:
We have been using padding instead of margin.

Also what if you removed this whole .section wrapper and added padding-top + padding-bottom to .sectionDivider?

Alternative solution would be to remove .sectionDivider,  add border-bottom, padding-top, padding-bottom to .section and to :last-type-of .section add border-bottom: none

Reply:
I sort of like the idea that there's no need for a section divider. Now the sectiondividers are just bolted in and don't really mean anything.. CSS selectors could probably handle this.